systemtest/main_test.go

```diff
@@ -24,11 +24,16 @@ import (
 )
 
 func TestMain(m *testing.M) {
+	log.Println("INFO: starting stack containers...")
 	if err := StartStackContainers(); err != nil {
 		log.Fatal(err)
 	}
+	log.Println("INFO: done.")
+	log.Println("INFO: setting up fleet...")
 	if err := Fleet.Setup(); err != nil {
 		log.Fatal(err)
 	}
+	log.Println("INFO: done.")
+	log.Println("INFO: running system tests...")
```

Comment:
Added these logging statements since they looked like nice output to have, but I can remove them from the PR since it's not strictly tied to this change.

Reply:
Looks good. Maybe drop the "done" logs, as they're implied by the immediately following logs.